pyproject.toml

```diff
@@ -28,7 +28,7 @@ requires-python = ">= 3.11, < 4"
 dependencies = [
   "typing_extensions >= 4.6.1, < 5",
   "grpc-stubs == 1.53.0.5",
-  "frequenz-api-reporting >= 0.2.0, < 0.5",
+  "frequenz-api-reporting >= 0.2.0, < 0.6",
```

Comment:
The lower version needs to be updated where `MetricConnections` was introduced, otherwise this fails.